go/vt/vtgate/executor_test.go

```diff
@@ -830,6 +831,20 @@ func TestExecutorShow(t *testing.T) {
 		t.Errorf("Got: %v. Want: %v", lastQuery, wantQuery)
 	}
 
+	// Set desitation keyspace in session
+	session.TargetString = KsTestUnsharded
+	_, err = executor.Execute(context.Background(), "TestExecute", session, "show create table unknown", nil)
+	if err != nil {
```

Comment:
nit: `require.NoError(t, err)`

Reply:
Addressed. Will change the rest to reflect this requirement in a separate PR